install/tools/ipa-acme-manage

```diff
@@ -0,0 +1,8 @@
+#!/bin/python -I
```

Comment:
This file should not be included, just the .in version.

Reply:
Thanks; fixed in next push.